### PR TITLE
Absolute values missing in benchmarks

### PR DESCRIPTION
**Pull request by ThomasCeulemans**
When computing the relative error in any benchmark, one needs to apply an absolute value before taking the maximum. This was missing in quite a few files.
Some benchmark reference values are changed accordingly.

### CODE REVIEW

**Comment by ThomasCeulemans** [comment]
As the accuracy of the evaluation of exponential has been lowered ec1151130fc4dbdbcb7832b564c5e1553bb25ce5, the benchmark results should first be checked before merging this.

**Comment by FredDeCeuster** [comment]
@ThomasCeulemans you can merge this after you checked the potential accuracy issue with the exponential.

**Review by FredDeCeuster — APPROVED** [review]
Fine, after you checked the potential issue with the accuracy of the exponentials.

**Pull request merged**
